misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JBallerinaDebugServer.java

```diff
@@ -526,6 +533,23 @@ void stepOut(int threadId) {
                     CompletionItem[] completions = getCompletions(visibleEntries);
                     completionsResponse.setTargets(completions);
                 }
+
+                if (resolverNode.isPresent() && resolverNode.get().kind() == SyntaxKind.REMOTE_METHOD_CALL_ACTION
+                        && resolverNode.get() instanceof RemoteMethodCallActionNode) {
+                    RemoteMethodCallActionNodeContext remoteMethodCallActionNodeContext =
+                            new RemoteMethodCallActionNodeContext();
+                    List<CompletionItem> completions = remoteMethodCallActionNodeContext
+                            .getCompletions(completionContext, ((RemoteMethodCallActionNode) resolverNode.get()));
+                    completionsResponse.setTargets(completions.toArray(new CompletionItem[0]));
+                }
+
+                if (resolverNode.isPresent() && resolverNode.get().kind() == SyntaxKind.ASYNC_SEND_ACTION
+                        && resolverNode.get() instanceof AsyncSendActionNode) {
```

Comment:
Lets remove the `instanceof` check and refactor these if conditions into switch cases

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/35371/commits/91c7f2f068c4270534d722eb7832d328afea4a7c